apps/web/components/Organization/OrganizationAlert/OrganizationAlert.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react'
+import {
+  AlertBanner,
+  AlertBannerVariants,
+  Box,
+  GridColumn,
+  GridContainer,
+  GridRow,
+  ResponsiveSpace,
+} from '@island.is/island-ui/core'
+import { AlertBanner as AlertBannerSchema } from '@island.is/web/graphql/schema'
+import { linkResolver, LinkType } from '@island.is/web/hooks'
+import { stringHash } from '@island.is/web/utils/stringHash'
+import Cookies from 'js-cookie'
+
+interface OrganizationAlertProps {
+  alertBanner: AlertBannerSchema
+  centered?: boolean
+  marginTop?: ResponsiveSpace
+}
+
+interface CenterContainerProps {
+  marginTop?: ResponsiveSpace
+}
+
+const CenterContainer: React.FC<CenterContainerProps> = ({
+  children,
+  marginTop,
+}) => {
+  return (
+    <GridContainer>
+      <GridRow marginTop={marginTop}>
+        <GridColumn span={['12/12', '12/12', '12/12', '12/12', '11/12']}>
+          {children}
+        </GridColumn>
+      </GridRow>
+    </GridContainer>
+  )
+}
+
+export const OrganizationAlert = ({
+  alertBanner,
+  centered,
+  marginTop,
+}: OrganizationAlertProps) => {
+  const alertBannerId = `alert-${stringHash(JSON.stringify(alertBanner))}`
+  const [shouldShowAlert, setShouldShowAlert] = useState<boolean>(
+    !Cookies.get(alertBannerId) && alertBanner.showAlertBanner,
+  )
+
+  if (!shouldShowAlert) return null
+
+  const link = {
+    ...(!!alertBanner.link &&
+      !!alertBanner.linkTitle && {
+        href: linkResolver(alertBanner.link.type as LinkType, [
+          alertBanner.link.slug,
+        ]).href,
+        title: alertBanner.linkTitle,
+      }),
+  }
+
+  const handleDismiss = () => {
+    if (alertBanner.dismissedForDays === 0) return
+    Cookies.set(alertBannerId, 'hide', {
```

Comment:
would it make sense to have something higher up in the state tree deciding whether or not to show this banner?

eg. how would you reuse the logic for another type of alert?

Would local storage be more appropriate for storing this kind of setting? Rather than a cookie.

Reply:
I used a cookie since that's what is used for the homepage of island.is, line 296 in apps/web/layouts/main.tsx
![image](https://user-images.githubusercontent.com/43557895/151821973-92ada821-2e53-48e6-8507-d01266d0d266.png)

There's a specific functionality for how long the dismiss is active so I did the same thing that was already in the code.

Reply:
And I decided not to have this state higher up since you might've noticed that I could have skipped using the OrganizationAlert component completely and only used the AlertBanner component which would have bloated up the OrganizationWrapper component, so that's my reasoning.

Reply:
makes sense